src/main/java/io/github/thebusybiscuit/slimefun4/implementation/items/multiblocks/miner/OreDictionary16.java

```diff
@@ -19,15 +19,13 @@ class OreDictionary16 extends OreDictionary14 {
     @Override
     @ParametersAreNonnullByDefault
     public @Nonnull ItemStack getDrops(Material material, Random random) {
-        switch (material) {
-            case NETHER_GOLD_ORE:
+        return switch (material) {
+            case NETHER_GOLD_ORE ->
                 // In 1.16, breaking nether gold ores should get gold nuggets
-                return new ItemStack(Material.GOLD_NUGGET, 2 + random.nextInt(4));
-            case ANCIENT_DEBRIS:
-                return new ItemStack(Material.ANCIENT_DEBRIS);
-            default:
-                return super.getDrops(material, random);
-        }
+                new ItemStack(Material.GOLD_NUGGET, 2 + random.nextInt(4));
```

Comment:
This does not need to be on a new line. Now the comment should be able to be moved above the case?